Jenkinsfile

```diff
@@ -45,8 +45,13 @@ pipeline {
 
         stage('Archive') {
             steps {
-                // Store the artifacts of the entire build
-                archive "**/*"
+                // Only store the HTML output in a directory with the same name
+                // as the branch it is on
+                sh "rm -rf env.BRANCH_NAME"
```

Comment:
Don't you need to use the "${}" syntax for that? Otherwise it'll be just the string literal which is not what you want.

Reply:
Ah yes, probably!

Reply:
Also branch names can have characters like `/` which have special meaning for filesystems.